cli/script/command-parser.ts

```diff
@@ -595,7 +596,8 @@ function createCommand(): cli.ICommand {
                     patchCommand.appName = arg1;
                     patchCommand.deploymentName = arg2;
                     patchCommand.label = argv["label"];
-                    patchCommand.description = argv["description"] ? backslash(argv["description"]) : "";
+                    patchCommand.description = argv["description"] ? backslash(argv["description"]) : argv["description"];
```

Comment:
add a comment saying that this is special case since all options for patch are optional, this needs to be null if not defined? May be also set explicitly null instead of same value ` : null; `?
Other places we set description to empty string.